lib/discharger/task.rb

```diff
@@ -265,6 +265,7 @@ def define
           exit if input.chomp.match?(/^x/i)
 
           tasker["reissue:finalize"].invoke
+          tasker["reissue:clear_fragments"].invoke
```

Comment:
The `reissue` task should do this https://github.com/SOFware/reissue/blob/66c997e7a785484ade155dcdd75e4d59859213bb/lib/reissue/rake.rb#L139